libraries/botbuilder-core/package.json

```diff
@@ -27,6 +27,7 @@
     }
   },
   "dependencies": {
+    "@azure/ms-rest-js": "1.9.1",
```

Comment:
I don't think this package actually imports anything from `@azure/ms-rest-js` right?

Reply:
If I remove that, I get this error while building
![image](https://user-images.githubusercontent.com/18757147/133260076-cfdf349c-b6aa-4eae-94de-a80941d20717.png)

Reply:
I'll push a quick commit fixing the type export and dependencies, it's a little subtle so easier to show than explain :)